packages/jss-preset-default/.size-snapshot.json

```diff
@@ -1,13 +1,13 @@
 {
   "dist/jss-preset-default.js": {
-    "bundled": 67203,
-    "minified": 23226,
-    "gzipped": 6969
+    "bundled": 30500,
```

Comment:
I would encourage the usage if [size-limit](https://github.com/ai/size-limit) over having to commit the new values every time a pull request is done. it's wasteful.

Reply:
size limit has also a fixed limit, which you have to fix if you increase, if you make it too flexible - than you don't control the size. I think the point of it is to make you aware of every size change on   for every change

Reply:
also we have different numbers here, which might behave differently, its interesting to see how minified/gziped size changes compared to raw, those numbers are not linear

Reply:
well, they only need to run yarn build, thats it, if not, we can as well do it, its not too bad so far, but if it gets annoying we can add a git hook cc @TrySound